docs/imagecreator/developer-guide.md

```diff
@@ -0,0 +1,42 @@
+---
+parent: Image Creator
+nav_order: 5
+---
+
+# Developers guide
+
+## Prerequisites
+
+- Golang
+- [Trivy](https://github.com/aquasecurity/trivy/releases/latest) for license scanning
+
+For other package dependencies, see [Getting started guide](./quick-start/quick-start-binary.md)
+
+## Build Image Creator binary
+
+Run:
+
+```bash
+sudo make -C ./toolkit go-imagecreator
+```
```

Comment:
Code blocks (and lists) should have an empty line before and after. This ensures maximum compatibility with Markdown parsers.

I like to use the `markdownlint` VSCode extension.